test/spec/modules/mygaruIdSyestem_spec.js

```diff
@@ -0,0 +1,41 @@
+import { mygaruIdSubmodule } from 'modules/mygaruIdSystem.js';
+import { server } from '../../mocks/xhr';
+
+describe('MygaruID module', function () {
+  it('should respond with async callback and get valid id', async () => {
```

Comment:
please add additional coverage tests for when your server responds with errors or empty responses. As well, what is the expected id value data type and length? in your docs it is closer to: `vatfS0UEgXShEqYuQy0lpzWl7rSuKeNs0U8C8841KoYbFUiyvTpS`

Reply:
84 symbols +-